pkgs/tools/misc/fdupes/default.nix

```
@@ -1,17 +1,21 @@
-{ stdenv, fetchFromGitHub }:
+{ stdenv, fetchFromGitHub, autoconf, automake, ncurses, pcre2, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "fdupes";
-  version = "1.6.1";
+  version = "2.1.1";
 
   src = fetchFromGitHub {
     owner = "adrianlopezroche";
     repo  = "fdupes";
     rev   = "v${version}";
-    sha256 = "19b6vqblddaw8ccw4sn0qsqzbswlhrz8ia6n4m3hymvcxn8skpz9";
+    sha256 = "1c5hv7vkfxsii1qafhsynzp9zkwim47xkpk27sy64qdsjnhysdak";
   };
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  nativeBuildInputs = [ autoreconfHook autoconf automake ncurses pcre2 ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook ];
  
  buildInputs = [ ncurses pcre2 ];
```

`autoconf` and `automake` can go. `ncurses` and `pcre2` should be `buildInputs`.

---

fixed, thanks